site-templates/resources/templates/base-tophat.html

```diff
@@ -5,7 +5,7 @@
       id="earthdata-tophat-script"
       data-show-fbm="true"
       data-show-status="true"
-      data-status-api-url="https://status.earthdata.nasa.gov/api/v1/notifications"
+      data-status-api-url={% if app-environment = "prod" %}"https://status.earthdata.nasa.gov/api/v1/notifications"{% elif app-environment = "uat" %}"https://status.uat.earthdata.nasa.gov/api/v1/notifications"{% elif app-environment = "sit" %}"https://status.sit.earthdata.nasa.gov/api/v1/notifications"{% else %}"https://status.earthdata.nasa.gov/api/v1/notifications"{% endif %}
```

Comment:
I didn't see any other template which used an if statement, should this be handled in code and dropped in as what ever URL the sever thinks it should be?

Reply:
Made the url a variable that just gets referenced here, no if statement involved